java/runtime/src/main/java/org/ray/runtime/MultiRuntimeHelper.java

```diff
@@ -0,0 +1,19 @@
+package org.ray.runtime;
+
+import org.ray.api.id.UniqueId;
+import org.ray.runtime.task.TaskExecutor;
+import java.util.HashMap;
+
+public class MultiRuntimeHelper {
+
+  private static HashMap<UniqueId, TaskExecutor> taskExecutors = new HashMap<>();
```

Comment:
It's better to put the map into `RayMultiWorkerNativeRuntime`. In the `get` method, we can find the runtime instance with `((RayMultiWorkerNativeRuntime)Ray.internal()).getRuntime(workerId)`. Basically, the main purpose of making the `get` method static is that you don't need to get a reference of the `MultiRuntimeHelper` instance in JNI.